src/pages/gettingstarted/angular.hbs

```diff
@@ -0,0 +1,368 @@
+          your own icon set, you need to import an svg that contains symbols for the ids referenced on the icon page. This symbol file
+          should occur in the DOM before the first use element.</p>
+      </li>
+    </ul>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Spark Extras</h2>
+
+    <div class="drizzle-o-ContentGrouping">
+      <p>
+          The Spark Angular Extras components rely on the Spark Core
+          Angular npm package and the SparkCoreAngularModule importation documented above.
+          Make sure that is done in order to use the extras components.
+          If your baseURL if not set in your Typescript config file then
+          you will need the additional './' prepended to the path
+          for the module imports.
+      </p>
+
+      <ul class="drizzle-b-List drizzle-b-List--nested sprk-u-mbm">
+        <li class="sprk-u-mvm">
+          <p>
+              Install the npm package for the Spark Extra that you need.
+          </p>
+          <div class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+            npm install --save-dev @sparkdesignsystem/spark-extras-angular-[package-name]</code>
+          </div>
+        </li>
+        <li class="sprk-u-mvm">
+          <p>You'll need to import the NgModule provided.</p>
+          <div class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+            import { Spark[PatternName]Module } from '@sparkdesignsystem/spark-extras-angular-[pattern-name]';
+            </code>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Browser Support</h2>
+    <p>Spark supports the following browsers (current version and 1 prior):</p>
+    <ul class="drizzle-b-List drizzle-b-List--spacing-s drizzle-b-List--nested">
+      <li>Google Chrome</li>
+      <li>Google Chrome (Mobile)</li>
+      <li>Mozilla Firefox</li>
+      <li>Mozilla Firefox (Mobile)</li>
+      <li>Microsoft Edge</li>
+      <li>Apple Safari</li>
+      <li>Apple Safari (Mobile)</li>
+    </ul>
+    <p>Spark also supports the following:</p>
+    <ul class="drizzle-b-List drizzle-b-List--spacing-s drizzle-b-List--nested">
+      <li>Microsoft Internet Explorer 11</li>
+    </ul>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+
+    <div class="drizzle-o-ContentGrouping">
+      <h2 class="drizzle-b-h2">Change Workflow and Contribution Guide</h2>
+      <p>If you are interested in contributing to Spark, please read our full
+        <a class="drizzle-b-Link" href="/docs/change-workflow">change
+          workflow</a> and <a class="drizzle-b-Link" href="/docs/contributions">
+          contribution guide</a> to get started.</p>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h3 class="drizzle-b-h3">Code Style Standards</h3>
```

Comment:
i think we should move the code style stuff to the contributions page

Reply:
Removed from html and angular doc pages, moved to contributions.